csharp/test/Ice/maxDispatches/AllTests.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) ZeroC, Inc.
+
+#nullable enable
+
+namespace Ice.maxDispatches;
+
+internal class AllTests : global::Test.AllTests
+{
+    internal static async Task allTests(global::Test.TestHelper helper)
+    {
+        Communicator communicator = helper.communicator();
+        string proxyString = $"test: {helper.getTestEndpoint()}";
+        Test.TestIntfPrx p = Test.TestIntfPrxHelper.createProxy(communicator, proxyString);
+
+        string proxyStringMax10 = $"test: {helper.getTestEndpoint(1)}";
+        Test.TestIntfPrx pMax10 = Test.TestIntfPrxHelper.createProxy(communicator, proxyStringMax10);
+
+        string proxyStringMax1 = $"test: {helper.getTestEndpoint(2)}";
+        Test.TestIntfPrx pMax1 = Test.TestIntfPrxHelper.createProxy(communicator, proxyStringMax1);
+
+        string proxyStringSerialize = $"test: {helper.getTestEndpoint(3)}";
+        Test.TestIntfPrx pSerialize = Test.TestIntfPrxHelper.createProxy(communicator, proxyStringSerialize);
+
+        await testMaxDispatches(p, 100, helper.getWriter());
+        await testMaxDispatches(pMax10, 10, helper.getWriter());
+        await testMaxDispatches(pMax1, 1, helper.getWriter());
+
+        // Serialize does not limit dispatches with "true" AMD.
```

Comment:
This test shows that with AMD, xxx.ThreadPool.Serialize=1 does not limit concurrent dispatches.

I am in favor of removing this Serialize property / implementation. I don't find it useful now that we have MaxDispatches.